[
    {
        "content": "<p>Hi guys, i have a problem with my runnable jar file.</p>\n<p>When i run it on a VM, i get the error message :<br>\nat ca.uhn.fhir.rest.client.exceptions.NonFhirResponseException.newInstance(NonFhirResponseException.java:63)<br>\n        at ca.uhn.fhir.rest.client.BaseClient$ResourceResponseHandler.invokeClient(BaseClient.java:483)<br>\n        at ca.uhn.fhir.rest.client.BaseClient$ResourceResponseHandler.invokeClient(BaseClient.java:441)<br>\n        at ca.uhn.fhir.rest.client.BaseClient.invokeClient(BaseClient.java:320)<br>\n        at ca.uhn.fhir.rest.client.GenericClient$BaseClientExecutable.invoke(GenericClient.java:741)<br>\n        at ca.uhn.fhir.rest.client.GenericClient$TransactionExecutable.execute(GenericClient.java:2377)</p>\n<p>The error occurs when my code tries to transfer the fhir bundle to the server:<br>\nIGenericClient client = ctx.newRestfulGenericClient(\"<a href=\"http://hapi.fhir.org/baseDstu3\" target=\"_blank\" title=\"http://hapi.fhir.org/baseDstu3\">http://hapi.fhir.org/baseDstu3</a>\");<br>\nBundle resp = client.transaction().withBundle(bundle).execute(); &lt;----------------- at this line</p>\n<p>It works fine on my computer, but not on the VM. At first i thought it has something to do with missing libraries or whatever, but part of the code works fine on the VM.</p>\n<p>EDIT: the problem was, i tried to access a DSTU3 server<br>\nMy code is DSTU2</p>",
        "id": 173171434,
        "sender_full_name": "Ma Sk",
        "timestamp": 1565752330
    }
]